examples/NASM/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2017 Zhuhao Wang
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-NASM)
+
+hunter_add_package(NASM)
+
+find_program(NASM_EXECUTABLE
```

Comment:
I think we need `HINTS` with `${NASM_ROOT}/bin` and `NO_DEFAULT_PATH`:
* https://cmake.org/cmake/help/latest/command/find_program.html